synthtool/gcp/templates/ruby_library/AUTHENTICATION.md

```diff
@@ -0,0 +1,179 @@
+# Authentication
+
+In general, the {{ metadata["package_name"] }} library uses [Service
+Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
+credentials to connect to Google Cloud services. When running on Compute Engine
```

Comment:
Should we expand this to something like "When running within [Google Cloud Platform environments](#google-cloud-platform-environments)"

Reply:
Done.